tests/test_policy.py

```diff
@@ -229,7 +229,8 @@ def test_resource_permissions(self):
                 found = bool(perms)
                 if not isinstance(perms, (list, tuple, set)):
                     found = False
-
+                if "webhook" in n:
```

Comment:
Why do you use `in` instead of `==`?

Reply:
Thanks, good catch - I have no idea why I misunderstood what `n` was.